### DISCUSSION TITLE
Docker command libnvidia-ml.so.1 load failed

### DISCUSSION THREAD

**Discussion by ramonemiliani93** [Project-MONAI/MONAILabel#788] [Q&A] [answered]
Hi! I am trying to run the Docker command listed on the installation guide:
```bash
docker run --gpus all --rm -ti --ipc=host --net=host projectmonai/monailabel:latest
```
But it fails and I get the following error:
```
Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: Running hook #0:: error running hook: exit status 1, stdout: , stderr: nvidia-container-cli: initialization error: load library failed: libnvidia-ml.so.1: cannot open shared object file: no such file or directory: unknown.
```
Are there any prerequisites for running the Docker container? The `--gpus` flag feels weird and may be causing the problem. I just want to spin up a version of the annotation platform on my MacBook (no GPU).

**Comment by ramonemiliani93** [comment, marked as answer]
Just removing the flag works 👌

**Reply by SachidanandAlle** [reply]
Docker needs gpu..  and gpu is preferred to run monailabel actions
